scripts/check-migration-numbering.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# The directory containing migration files.
+migrations_path="tapdb/sqlc/migrations"
+
+# Check if the directory exists
+if [ ! -d "$migrations_path" ]; then
+  echo "Directory $migrations_path does not exist."
+  exit 1
+fi
+
+# Get all unique prefixes (e.g., 000001, always 6 digits) from .up.sql files.
+prefixes=($(ls "$migrations_path"/*.up.sql 2>/dev/null | \
```

Comment:
Super-nit; a simpler command:

`ls -1 tapdb/sqlc/migrations/*.up.sql | cut -d'_' -f1 | cut -d'/' -f4 | sort -n`

Reply:
That seems a bit brittle if someone uses unexpected characters in the file name...